image/image.go

```diff
@@ -228,3 +311,37 @@ func createRuntimeBundle(w walker, dest, refName, rootfs string) error {
 
 	return json.NewEncoder(f).Encode(spec)
 }
+
+// filertManifest returns a filtered list of manifests
+func filterManifest(w walker, Manifests []v1.Descriptor, platform []string) ([]*manifest, error) {
+	var manifests []*manifest
+
+	if len(Manifests) == 0 {
+		fmt.Println("warning: no manifests found")
+		return manifests, nil
+	}
+
+	if len(platform) != 2 {
```

Comment:
Either this should be checked far earlier (when you do the splitting) or you should only split it here. I would prefer if you just passed the string down to `filterManifest` and then you verify the arguments here.